proto/metapb.proto

```diff
@@ -73,6 +83,7 @@ message Region {
     // The field is only used by PD and should be ignored otherwise.
     // If encryption_meta is empty (i.e. nil), it means start_key and end_key are unencrypted.
     encryptionpb.EncryptionMeta encryption_meta = 6;
+    Buckets buckets = 7;
```

Comment:
I was thinking the buckets are queries and updated in a standalone background fashion. For example, client can use stale buckets to split queries without actually depending on the information from Region.

Reply:
The related part of RFC is in https://github.com/tikv/rfcs/blob/8bd15f24fd01488a6d639d5a13e7876ddb3159da/text/0082-dynamic-size-region.md?plain=1#L104-L111.

Reply:
Is there any problem of using existing GetRegion API to get the latest version of buckets? Client can still have the logic described in the RFC. 
To me the region's other fields are very minimal. Having a specific buckets API is not necessary.

Reply:
For reporting bucket stats, we will use stream API which will be defined in Minhua's PR

Reply:
There are 3 reasons:
1. buckets change frequently, they don't need to be persisted, while region must be persisted;
2. if buckets are associated with stats, then we don't need to do anything specific for matching buckets with stats;
3. unlike regions, buckets are not available all the time. For example, if a peer is elected as a new leader, or split worker is busy, or the node is restarted, then buckets may not be available for quite a long time.

Given the difference between buckets and regions, handle them respectively is more flexible and easy to implement correctly.